JetAnalyzers/bin/jet_response_analyzer_x.cc

```diff
@@ -1478,6 +1478,12 @@ int main(int argc,char**argv)
 
         if (nrefmax>0) JRAEvt->nref = std::min((int)JRAEvt->nref,nrefmax);
         for (unsigned char iref=0;iref<JRAEvt->nref;iref++) {
+	  /*      
```

Comment:
Same comment about commented code.

Reply:
Okay, I'll remove the commented out code.